src/Resources/config/listener.yml

```diff
@@ -6,6 +6,7 @@
 #
 # - 30: ContainerScopeListener
 # - 28: RefererIdListener
+# - 27: InsecureInstallationListener
```

Comment:
> The priorities of the following listeners must be lower than the one of the Symfony route listener (defaults to 32).

And we want to give users and developers a chance to register own listeners in between one of our listeners, therefore we keep one free number in between.